### PR TITLE
PR-506 => develop

### PR DESCRIPTION
**Pull request by niklasIsento**

### CODE REVIEW

**Comment by niklasIsento** [comment]
- [ ] All acceptance criteria have been implemented.

- [ ] Tester feedback has been incorporated.

- [ ] If necessary, a documentation has been created/ code-comments written.

- [ ] At least one successful code review has been conducted.

- [ ] If necessary, an appointment for knowledge transfer has been arranged.

- [ ] Dependencies on other repositories have been checked and implemented if necessary.

- [ ] At least one successful setup-test on a virtual machine was done.

- [ ] At least one successful setup-test on a raspberry pi/ physical pib was done.

File: motors/motor_control.py [hunk 1]
```diff
@@ -218,6 +225,9 @@ def joint_trajectory_callback(self, msg):
                                         if motor.name == "shoulder_vertical_left" or motor.name == "shoulder_vertical_right":
```
Comment (JaHe-isento) [inline motors/motor_control.py:225]: This check should be made unnecessary by this story. Disable the initialized values in the database and the motor list instead.

File: motors/motor_control.py [hunk 2]
```diff
@@ -113,33 +118,33 @@ def __init__(self):
...
+                        Motor("thumb_right_stretch", self.servo1, [4], True, 1, False),
+                        Motor("index_right_stretch", self.servo1, [5], True, 1, False),
+                        Motor("middle_right_stretch", self.servo1, [6], True, 1, False),
+                        Motor("ring_right_stretch", self.servo1, [7], True, 1, False),
+                        Motor("pinky_right_stretch", self.servo1, [8], True, 1, False),
+                        Motor("upper_arm_left_rotation", self.servo2, [5], True, 2, False), 
+                        Motor("elbow_left", self.servo2, [6], True, 2, False),
+                        Motor("lower_arm_left_rotation", self.servo2, [7], True, 2, False),
+                        Motor("wrist_left", self.servo1, [2], True, 2, False),
+                        Motor("shoulder_vertical_left", self.servo3, [7, 9], True, 2, False),
```
Comment (JaHe-isento) [inline motors/motor_control.py:140]: The initialisation value for shoulder_vert should probably be set to true

File: motors/motor_control.py [hunk 3]
```diff
@@ -113,33 +118,33 @@ def __init__(self):
...
+                        Motor("shoulder_horizontal_left", self.servo3, [0], True, 2, False),
+                        Motor("upper_arm_right_rotation", self.servo3, [1], True, 3, False),
+                        Motor("elbow_right", self.servo3, [2], True, 3, False),
+                        Motor("lower_arm_right_rotation", self.servo3, [3], True, 3, False),
+                        Motor("wrist_right", self.servo3, [4], True, 3, False), 
+                        Motor("shoulder_vertical_right", self.servo3, [5, 8], True, 3, False),
```
Comment (JaHe-isento) [inline motors/motor_control.py:146]: The initialisation value for shoulder_vert should probably be set to true

**Pull request merged**